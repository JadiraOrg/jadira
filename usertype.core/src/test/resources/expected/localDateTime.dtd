<!ELEMENT dataset (
    LOCALDATETIME*)>

<!ELEMENT LOCALDATETIME EMPTY>
<!ATTLIST LOCALDATETIME
    ID CDATA #REQUIRED
    LOCALDATETIME CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

