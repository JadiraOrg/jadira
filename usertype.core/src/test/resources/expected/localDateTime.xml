<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "localDateTime.dtd">
<!--

  Copyright 2010, 2011 Christopher Pheby

  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at

      http://www.apache.org/licenses/LICENSE-2.0

  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.

 -->
<dataset>
  <LOCALDATETIME ID="0" LOCALDATETIME="2004-02-25 12:11:10.0" NAME="test_0"/>
  <LOCALDATETIME ID="1" LOCALDATETIME="1980-03-11 13:12:11.0" NAME="test_1"/>
</dataset>